NBA Live 96

```diff
@@ -0,0 +1,94 @@
+Cuonzo Martin
+Alan Henederson
+Amory Wong
+Andrew DeClercq
+Anthony Pelle
+Antonio McDyess
+Bill Laimbeer
+Bill Russell
+Bill Walton
+Bob Cousy
+Bob Sura
+Brent Barry
+Brian Krause
+Bryant Reeves
+C.Williamson
+Calvin Murphy
+Charles Barkley
+Cherokee Parks
+Chris Carr
+Constantin Popa
+Corey Alexander
+Cuonzo Martin
+Damon Stoudamire
+Dave Cowens
+David Thompson
+David Vaughn
+Dejan Bodiroga
+Don Reid
+Donny Boyce
+Donny Marshall
+Dragan Tarlac
+Dwayne Whitfield
+Ed O'Bannon
+Elgin Baylor
+Elvin Hayes
+Eric Snow
+Eric Williams
+Erik Meek
+Frankie King
+Fred Hoiberg
+Gary Shaw
+Gary Trent
+George Banks
+George Gervin
+George Mikan
+George Zidek
+Greg Ostertag
+Isiah Thomas
+James Worthy
+Jason Caffey
+Jerome Allen
+Jerry Stackhouse
+Jerry West
+Jimmy King
+Joe Smith
+John Havlicek
+Julius Erving
+Junior Burrough
+K.Abdul-Jabbar
+Ken Thurston
+Kevin Garnett
+Kevin McHale
+Kurt Thomas
+Larry Bird
+Lawrence Moten
+Loren Meyer
+Lou Roe
+Magic Johnson
+Mario Bennett
+Mark Davis
+Martin Lewis
+Michael Finley
+Michael Jordan
+Mike Kiernan
+Nate Archibald
+Nate Thurmond
+Oscar Robertson
+Pete Maravich
+R.Childress
+Rashard Griffith
+Rasheed Wallace
+Rick Barry
+Shawn Respert
+Sherell Ford
+Stanley Chow
+Terrence Rancher
+Theo Ratliff
+Travis Best
+Troy Brown
+Tyus Edney
+Walt Frazier
+Wilt Chamberlain
+Shaquille O'Neal
+The rest of the 1996-1997 NBA Roster

```

Comment:
To match formatting with NBA Street V3, please change this to "1996-97 NBA Season roster"